tools/docker-compose/bootstrap_development.sh

```diff
@@ -40,7 +40,7 @@ echo "Admin password: ${DJANGO_SUPERUSER_PASSWORD}"
 awx-manage create_preload_data
 awx-manage register_default_execution_environments
 
-awx-manage provision_instance --hostname="$(hostname)" --node_type="$MAIN_NODE_TYPE"
+awx-manage provision_instance --hostname="$(hostname)" --node_type="$MAIN_NODE_TYPE" --listener_port=2222
```

Comment:
This concerns me a little bit about the expected practice. Does this need to be provided for VM installs in general? I don't get why we didn't need this before, but would now.